source/overloading.tex

```diff
@@ -3816,114 +3779,6 @@
 The prefix and postfix decrement operators
 \tcode{-{-}}
 are handled analogously.
```

Comment:
We never define what a "decrement operator function" is. Perhaps:

> A *decrement operator function* is handled analogously to an increment operator function, except for the operator `--` instead of `++`.

or something like that.

Reply:
I did:

A \defnadj{decrement}{operator function}
is a function named \tcode{\keyword{operator}\dcr}
and is handled analogously to an increment operator function.